src/store/reducers/devicesReducer.js

```diff
@@ -179,4 +206,16 @@ export const getDevices = createSelector(
   getEntities, getItems,
   (entities, items) => items.map(id => entities[id])
 );
+export const getDeviceModuleStatus = state => {
+  const deviceModuleStatus = getDevicesReducer(state).deviceModuleStatus
+  return deviceModuleStatus ? {
+    code: deviceModuleStatus.code,
+    description: deviceModuleStatus.description
+  }
```

Comment:
Watch the alignment here. Just like a normal if else block should align, so should the conditions in ? :

Reply:
fixed